lib/Runtime/Library/JavascriptBigInt.cpp

```diff
@@ -397,4 +414,183 @@ namespace Js
         return (leftBigInt->Compare(rightBigInt) == 0);
     }
 
+    // pbi1 += pbi2 assume pbi1 > pbi2
+    void JavascriptBigInt::AddAbsolute(JavascriptBigInt * pbi1, JavascriptBigInt * pbi2)
+    {
+        digit_t carryDigit = 0;
+        digit_t *pDigit1 = pbi1->m_digits;
+        digit_t *pDigit2 = pbi2->m_digits;
+        digit_t i = 0;
+
+        for (; i < pbi2->m_length; i++)
+        {
+            digit_t tempCarryDigit = 0;
+            if (pDigit2[i] > 0)
```

Comment:
Do you know if these `> 0` checks save a lot of time? If not, or if you're unsure, I'd prefer to omit them and make the code more compact.

Reply:
will remove